src/test/java/giis/eshopcontainers/e2e/functional/CatalogTests.java

```diff
@@ -0,0 +1,59 @@
+package giis.eshopcontainers.e2e.functional;
+
+import giis.eshopcontainers.e2e.functional.common.BaseLoggedClass;
+import giis.eshopcontainers.e2e.functional.common.ElementNotFoundException;
+import giis.eshopcontainers.e2e.functional.utils.Click;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import static giis.eshopcontainers.e2e.functional.utils.Shopping.*;
+
+class CatalogTests extends BaseLoggedClass {
+    @Test
+    @DisplayName("AddProductsToBasket")
+    void addProductsToBasket() throws ElementNotFoundException {
+        log.debug("Checking that the product buttons are disabled before the user logs-in");
+        WebElement productCupButton = driver.findElement(By.xpath("/html/body/div/div[3]/div[1]/form/input[1]"));
+        Assertions.assertEquals("esh-catalog-button is-disabled", productCupButton.getAttribute("class"), "Before Login the eShop product button was expected for being disabled and was enabled");
+        this.login();
+        WebElement loggedUser = driver.findElement(By.xpath("//*[@id=\"logoutForm\"]/section[1]/div"));
+        Assertions.assertEquals("alice", loggedUser.getText(), "The user logged is not the expected, the expected was alice and the obtained is " + loggedUser.getText());
```

Comment:
Why alice is checked to be the logged user if the login didn't specified the user? Should this check be part of the login method who knows the user that is bein logged?